store/conf/attrs/zimbra-attrs.xml

```diff
@@ -9598,4 +9598,102 @@ TODO: delete them permanently from here
   <desc>Search history entries older than this will not show up in search suggestions and will not be included in the count used for zimbraNumSearchesForSavedSearchPrompt or as part of GetSearchHistoryResponse/SearchSuggestResponse. If purging is enabled, these entries will be purged as part of a mailbox purge.</desc>
   <defaultCOSValue>365d</defaultCOSValue>
 </attr>
+
+<attr id="3023" name="zimbraIndexURL" type="string" cardinality="single" optionalIn="globalConfig,server,alwaysOnCluster" flags="serverInherited,serverPreferAlwaysOn" since="8.8.1">
+    <globalConfigValue>solr:http://localhost:7983/solr</globalConfigValue>
+    <desc>URL for accessing external indexing service.
+    First part of the URL before the first colon identifies the implementation Factory and should be registered by calling IndexStore.registerFactory(prefix, classname).
+    ZCS ships with two index Factory implementations:
+    1) SolrIndex identified by prefix "solr".
+    2) SolrCloudIndex identified by prefix "solrcloud".
+
+    If using solrcloud, the URL should be set to that of the ZooKeeper host managing the SOLR cluster.
+     </desc>
+</attr>
+
+<attr id="3024" name="zimbraIndexDeferredItemsFailureDelay" type="integer" cardinality="single" optionalIn="globalConfig,server,alwaysOnCluster" flags="serverInherited,serverPreferAlwaysOn" since="8.8.1">
+  <globalConfigValue>300</globalConfigValue>
+  <desc>If indexing of an item is failed. It can be tried after the delay specified in seconds</desc>
+</attr>
+
+<attr id="3025" name="zimbraIndexManualCommit" type="boolean" cardinality="single" optionalIn="globalConfig,server,alwaysOnCluster" flags="serverInherited,serverPreferAlwaysOn" since="8.8.1">
+  <globalConfigValue>FALSE</globalConfigValue>
+  <desc>When set to TRUE, server will commit changes to Solr after every update request. When set to FALSE, Solr commits will be performed according to Solr configuration. Set to TRUE for automated testing. Recommended production setting is FALSE.</desc>
```

Comment:
Yeah I agree. It may be better to remove this code entirely at this point, since I'm pretty sure the softAutoCommit approach provides the functionality we need. If not, we need to remove the dependency on real-time search entirely.